src/gui/wizard/owncloudwizard.cpp

```diff
@@ -201,6 +201,11 @@ void OwncloudWizard::slotCurrentPageChanged(int id)
     }
 
     setOption(QWizard::HaveCustomButton1, id == WizardCommon::Page_AdvancedSetup);
+    if (id == WizardCommon::Page_AdvancedSetup && _credentialsPage == _browserCredsPage) {
+        // For OAuth, disable the back button in the Page_AdvancedSetup because we don't want
+        // to re-open the browser.
+        button(QWizard::BackButton)->setEnabled(false);
```

Comment:
1) The QWizard already reset the enabled property on the buttons each time the page change.
2) Since the next page is the final one and its not possible to go back, either, we are fine.